src/clims/api/endpoints/work_units.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import
+
+import logging
+
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from sentry.api.bases.organization import OrganizationEndpoint
+
+from sentry.api.base import DEFAULT_AUTHENTICATION, Endpoint
+
+from clims.api.serializers.models.workunit import WorkUnitSerializer
+
+logger = logging.getLogger(__name__)
+
+
+class WorkUnitsEndpoint(OrganizationEndpoint):
```

Comment:
Add documentation that says what WorkUnit really is (instance of work on one "tracked object", usually Substance)